docker-compose.yml

```diff
@@ -1,18 +1,33 @@
 version: "3"
 services:
   mongo:
-    image: mongo:3.5.7
-    ports:
-      - 27017:27017
+    image: mongo:3.5.9
+    restart: always
+    expose:
+      - "27017"
 
   rabbitmq:
-    image: rabbitmq:3.6.9
-    ports:
-      - 5672:5672
+    image: rabbitmq:3.6.10
+    restart: always
+    expose:
+      - "5672"
+
+  # web:
+  #   build: web
+  #   restart: always
+  #   ports:
+  #     - 80:5000
+  #   depends_on:
+  #     - mongo
+  #   environment:
+  #     - FLASK_APP=server.py
+  #     - MONGO_DATABASE=whistleblower
+  #     - MONGO_URL=mongodb://mongo:27017/
```

Comment:
Do we really need this commented block committed? Apart from that LGTM. LMK what do you think and I merge this branch ; )

Reply:
just out of interest of releasing the code... would merge and open an issue to remove the comments...

Reply:
Changed as you asked.